Examples/LambdaFunctions/Sources/LambdaAPI/ProductService.swift

```diff
@@ -0,0 +1,104 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftAWSLambdaRuntime open source project
+//
+// Copyright (c) 2020 Apple Inc. and the SwiftAWSLambdaRuntime project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftAWSLambdaRuntime project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+import AWSDynamoDB
+import NIO
+
+public enum APIError: Error {
+    case invalidItem
+    case tableNameNotFound
+    case invalidRequest
+    case handlerNotFound
+}
+
+extension Date {
+    var iso8601: String {
+        let formatter = DateFormatter()
+        formatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"
+        formatter.timeZone = TimeZone(secondsFromGMT: 0)
+        return formatter.string(from: self)
+    }
+}
+
+public class ProductService {
+    
+    let db: DynamoDB
+    let tableName: String
+    
+    public init(db: DynamoDB, tableName: String) {
+        self.db = db
+        self.tableName = tableName
+    }
+    
+    public func createItem(product: Product) -> EventLoopFuture<DynamoDB.PutItemOutput> {
+        
+        var product = product
+        let date = Date().iso8601
+        product.createdAt = date
+        product.updatedAt = date
```

Comment:
can the date be stored in more native format like numeric or timestamp?

Reply:
074b7a0

Note that in DynamoDB the date can be saved as Number o String.
https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.NamingRulesDataTypes.html#HowItWorks.DataTypes